metisp/pymetis/src/pymetis/classes/inputs/base.py

```diff
@@ -161,4 +153,10 @@ def valid_frames(self) -> cpl.ui.FrameSet:
         Return a FrameSet containing all valid, used frames.
         This is abstract as it differes significantly for Single and Multiple Inputs.
         """
-        pass
\ No newline at end of file
+        pass
+
+    @classmethod
+    def _pretty_tags(cls) -> str:
+        """ Helper method to print `re.Pattern`s in man-page: remove named capture groups' names. """
+        return cls.tags().pattern
+        return re.sub(r"\?P<\w+>", "", cls.tags().pattern)
```

Comment:
Two return statements in a row seems like a mistake.

Reply:
Forgot about this from some late night debugging session. Right now I'd prefer the unmodified version, since it explicitly shows what the name of the parameter is -- and how it is propagated to outputs.